app/pulp/app/models/task.py

```diff
@@ -35,6 +35,30 @@ class ReservedResource(Model):
     worker = models.ForeignKey("Worker", on_delete=models.CASCADE, related_name="reservations")
 
 
+class WorkerManager(models.Manager):
+    def get_unreserved_worker(self):
+        """
+        Find an unreserved Worker
```

Comment:
Can the string 'Worker' be updated with the sphinx class path so it links to the actual `Worker` module?